internal/pkg/scorecard/scorecard.go

```diff
@@ -310,7 +312,63 @@ func runTests() ([]TestSuite, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to merge test suite results: %v", err)
 	}
-	return suites, nil
+	for _, suite := range suites {
+		// convert to ScorecardOutput format
+		// will add log when basic and olm tests are separated into plugins
+		pluginResults = append(pluginResults, TestSuitesToScorecardOutput([]TestSuite{suite}, ""))
+	}
+	// Cleanup built-in test resources before running plugins
+	if err := cleanupScorecard(); err != nil {
+		log.Errorf("Failed to cleanup resources: (%v)", err)
+	}
+	// Run plugins
+	pluginDir := viper.GetString(PluginDirOpt)
+	if dir, err := os.Stat(pluginDir); err != nil || !dir.IsDir() {
+		log.Warnf("Plugin directory not found; skipping plugin tests")
+	} else {
+		if err := os.Chdir(pluginDir); err != nil {
+			return nil, fmt.Errorf("failed to chdir into scorecard plugin directory")
+		}
+		// executable files must be in "bin" subdirectory
+		files, err := ioutil.ReadDir("bin")
+		if err != nil {
+			return nil, fmt.Errorf("failed to list files in %s/bin", pluginDir)
+		}
+		for _, file := range files {
+			cmd := exec.Command("bash", "-c", "./bin/"+file.Name())
```

Comment:
We need a shell to be able to start/execute a shell script. Since we support executing shell scripts, we need to start them through a shell. We could probably use `sh` as well, since the actual language for the shell script should be defined by the first line of the script (like `#!/bin/bash`), but I just assumed we would always run on a system that has bash.

Reply:
The shebang will take care of shell creation. I think we should not add any system-specific configuration to running binaries, even if `bash` is supported and present in virtually all systems.

Reply:
Huh. When I was first testing it, it wasn't working without bash. I guess those errors must've been due to something else because it seems to work now. Changed to execute directly.